src/tests/ftest/pool/evict_test.py

```diff
@@ -38,110 +39,249 @@ class EvictTest(TestWithServers):
 
     def test_evict(self):
         """
-        Test evicting a client to a pool.
+        Test evicting a client from a pool.
 
-        :avocado: tags=pool,poolevict,quick
+        :avocado: tags=pool,poolevict,quick,mjean
         """
-        size = self.params.get("size", '/run/tests/sizes/size1gb/*')
-        setid = self.params.get("setname", '/run/tests/setnames/validsetname/*')
-        connectperm = self.params.get("perms",
-                                      '/run/tests/connectperms/permro/*')
+        pool = []
+        container = []
+        cont_uuid = []
+        non_pool_servers = []
+        # Target list is configured so that the pools are across all servers
+        # except the pool under test is created on half of the servers
+        pool_tgt = [num for num in range(len(self.hostlist_servers))]
+        pool_tgt_ut = [num for num in range(int(len(self.hostlist_servers)/2))]
+        tlist = [pool_tgt, pool_tgt, pool_tgt_ut]
+        pool_servers = [self.hostlist_servers[:len(tgt)] for tgt in tlist]
+        # Create TestPool
+        for count in range(len(tlist)):
```

Comment:
(pylint-consider-using-enumerate) Consider using enumerate instead of iterating with range and len